recipes/libstempo/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "libstempo" %}
+{% set version = "2.3.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b3b6ed668b04c7ea445a4594e1366f97458da051adda25f1718b3f29cb0ca470
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - astropy
```

Comment:
I know `scipy` doesn't have a compiled API you'd be linking against, and I don't think `astropy` does either, so you *shouldn't* need them in `host`? Does that also apply to `tempo2`?

Reply:
Thanks. I've removed scipy and astropy from host. Tempo2 is compiled and linked to the cython compiled library within libstempo, so I think may be needed.

Reply:
👍